controllers/authController.js

```diff
@@ -0,0 +1,63 @@
+const users = require("../DB/FakeUsers");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const passport = require("passport");
+const JwtStrategy = require("passport-jwt").Strategy;
+const ExtractJwt = require("passport-jwt").ExtractJwt;
+
+const register = (req, res) => {
+    const user = {
+        id: Math.round(Math.random() * 1000),
```

Comment:
Probably need to double check if ID is unique

Reply:
> Probably need to double check if ID is unique

It is just example, I will delete this after using mongodb.

Reply:
ok, then pr can be merged, good job)